advocacy_docs/partner_docs/DBeaverGuide/05-UsingDBeaver.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Using DBeaver PRO'
+description: 'Walkthroughs of example usage scenarios'
+---
+Once DBeaver PRO is connected to an instance of EDB Postgres Advanced Server or EDB Postgres Extended, you can then access the capabilities of DBeaver PRO.
+
+
+# Sample User Scenarios
+
+!!! Note
+    The user scenarios shown are provided as example use cases of DBeaver PRO and EDB Postgres Advanced Server or EDB Postgres Extended.  They are not intended to show all functionality.
+
+## Connecting to the Database
+
+1. Launch DBeaver Pro.
+2. Double click the database connection you want to open.
+3. Enter the username and password and click `OK`.
+
+<p align="center">
+  <img src="Images/DBNavigate.png">
+</p>
+
+4. You are now connected to your database instance.
+
+## Viewing Tables and Data
+
+Using the appropriate SQL code, three tables (`AGENTS`, `CUSTOMERS`, and `ORDERS`) were created in the `edb` database and data was inserted to help visualize some basic functionality.  
+
+<p align="center">
+  <img src="Images/DBNavigate2.png">
+</p>
+
+1. In the Database Navigator, right click on your database in the example that is `edb`.
```

Comment:
right click on your database in the example that is `edb`.
change to
right click on your database. In this example that is `edb`.